src/views/product-landing/helpers/__tests__/make-heading-slug-scope.test.ts

```diff
@@ -0,0 +1,24 @@
+import { makeHeadingSlugScope } from '../'
+
+describe('makeHeadingSlugScope', () => {
```

Comment:
Set up some tests here to try to validate that `makeHeadingSlugScope` isn't too wild of an idea 😅